ludwig/visualize.py

```diff
@@ -1494,7 +1502,6 @@ def compare_performance(
         metric_names = metric_names_sets[0]
         for metric_names_set in metric_names_sets:
             metric_names = metric_names.intersection(metric_names_set)
-        metric_names.remove(LOSS)
```

Comment:
I'm not sure why this was here. The structure of the test statistics file changes from datatype to datatype, so maybe this is needed for a specific datatype. I suggest not to remove this but add an if instead

Reply:
I think this was a mistake on my part. Let me add this back in.